transport_nantes/mailing_list/templates/mailing_list/status/hidden_form.html

```diff
@@ -0,0 +1,22 @@
+<form method="post"
+    {% if unsubscribe %}
+        action="{% url 'mailing_list:status_unsubscribe' mailing_list.id %}"
+    {% else %}
+        action="{% url 'mailing_list:status_subscribe' %}"
+    {% endif %}>
+    {% csrf_token %}
+    <input type="hidden" id="user" name="user" value="{{user_id}}">
```

Comment:
Why include the user id in the form while you can retrieve it in the view ? Which you do next btw, there is not plus side, as so far as I'm aware, to add this id to the form ?